src/core/Akka.Remote/Serialization/PrimitiveSerializers.cs

```diff
@@ -78,6 +85,9 @@ public override object FromBinary(byte[] bytes, string manifest)
         /// <inheritdoc />
         public override string Manifest(object obj)
         {
+            if (_useNeutralPrimitives)
+                return obj.GetType().TypeQualifiedName();
```

Comment:
Need to emulate the old serializer behaviour, which is to embed the type name as manifest.
Related code:
https://github.com/akkadotnet/akka.net/blob/dev/src/core/Akka.Remote/MessageSerializer.cs#L62-L74